src/docfx/lib/git/GitUtility.cs

```diff
@@ -314,23 +311,23 @@ private static async Task CloneOrUpdate(string path, string url, IEnumerable<str
 
             try
             {
-                await ExecuteNonQuery(path, $"{httpConfig} fetch --tags --progress --update-head-ok {pruneSwitch} {depth} \"{url}\" {refspecs}", stderr: true);
+                ExecuteNonQuery(path, $"{httpConfig} fetch --tags --progress --update-head-ok {pruneSwitch} {depth} \"{url}\" {refspecs}");
             }
-            catch (InvalidOperationException ex) when (committishes.Any(rev => ex.Message.Contains(rev)))
+            catch (InvalidOperationException)
```

Comment:
removed `stderr` from `ProcessUtility.Execute`, we'll retry here in most cases

Reply:
what about the log info for debugging when the execution failed?

Reply:
`stderr` will be piped to output, so you can see that from the log